modules/apps/data-engine/data-engine-test/src/testFunctional/paths/DataEngineRenderer.path

```diff
@@ -55,6 +55,16 @@
 	<td>xpath=(//*[name()='label' or name()='legend'][normalize-space(text())='${key_fieldLabel}']/ancestor::div[contains(@class,'form-group')]//button[normalize-space()='${key_buttonName}'])[${index}]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>BUTTON_LABEL_INDEXED</td>
+	<td>xpath=(//div[./label[normalize-space(text())='${key_fieldLabel}']]//button[normalize-space()='${text}'])[${index}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECTOR_INPUT_INDEXED</td>
+	<td>xpath=(//label[normalize-space(text())='${key_fieldLabel}']/following-sibling::div//div[contains(@class,'input-group')]//input[not(contains(@type,'hidden'))])[${index}]</td>
```

Comment:
@guivilarim, the variables `text` and `index` are still missing the prefix `key_`, can you please add it?

Thanks